src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -109,6 +109,10 @@ private static function formatFileSize(string $path): string
         if (is_file($path)) {
             $size = filesize($path) ?: 0;
         } else {
+            if (!is_dir($path)) {
+                return 'n/a';
```

Comment:
the abstraction for "formatted file size" is any string, hence this looks appropriate to me